src/Routing/Router.php

```diff
@@ -987,8 +987,14 @@ public static function prefix($name, $params = [], $callback = null)
             $callback = $params;
             $params = [];
         }
-        $name = Inflector::underscore($name);
-        $path = '/' . $name;
+
+        if (empty($params['path'])) {
+            $path = '/' . Inflector::underscore($name);
+        } else {
+            $path = $params['path'];
+            unset($params['path']);
+        }
```

Comment:
Could we get a test added for this new option? Also the docblock should be updated.